modules/apps/collaboration/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/css/main.scss

```diff
@@ -148,6 +148,12 @@
 	}
 
 	.edit-entry {
+		.alloy-editor-placeholder:before {
```

Comment:
Hey Ambrin!

Try with this:

.ae-placeholder:not(:focus):before {
   content: attr(data-placeholder);
   position: absolute;
   transform: translateX(-50%);
}